AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Dependency/AuthorizationProviderAdapter+SignedInSession.swift

```diff
@@ -155,10 +155,12 @@ extension AuthorizationProviderAdapter {
             guard task.error == nil else {
                 if let urlError = task.error as NSError?, urlError.domain == NSURLErrorDomain {
```

Comment:
nit: consider mutually exclusive if/else block here to ensure you `return` from each condition